README.md

```diff
@@ -99,7 +99,7 @@ You can create the request also by simply calling:
 
 That command has a bit of automagic, it will:
 
-1. lookup the namespace and project of the current branch (or at least on the `github` 
+1. lookup the namespace and project of the current branch (or at least on the `github`
```

Comment:
could you move these unecessary modifications to a separate PR. I like nice clean small PRs that do one thing only.

Reply:
Okay. I think that change was done automatically by my editor and I didn't notice that. Totally agree - there is no need to have this unnecessary modification in this PR.